### ISSUE TITLE
Inculde cookie duration in tracking heuristic.

### ISSUE THREAD

**Issue by cowlicks** [EFForg/privacybadger#1545]
Once this is done, we should remove weatherzone.com.au from the CBL (#1543).

We should also revisit cookies set by things on the CBL to see what their duration is. And what this would fix.

**Comment by ghostwords** [comment]
Yep, we should probably account for short-term cookies and maybe also session cookies (https://github.com/EFForg/privacybadger/issues/1539#issuecomment-319172618).

Removing domains from the yellowlist is dicey as long as pre-2017.7.24 are still around (https://github.com/EFForg/privacybadger/issues/1474#issuecomment-314803154).

**Comment by cowlicks** [comment]
Since we've mistakenly tied privacy badger versions to cookieblock list behavior, now would be a good time to fix any problems with it. We can make new versions pull from a new URL. And we won't have to worry about breaking old versions.

**Comment by ghostwords** [comment]
Ignoring session cookies seems like the right thing to do given that session cookies are meant to expire at the end of the browsing session (apparently [Chrome preserves them](https://bugs.chromium.org/p/chromium/issues/detail?id=128513) when you set it to "Continue where I left off" though), which seems to make them much less effective for tracking versus cookies with (far-future) expiration dates.

I think it's worth visiting a bunch of sites (from error reports?) and logging what Badger learned to block because of session cookies alone. If it's all non-obviously-tracking domains, let's do it.

**Comment by bcyphers** [comment]
What's the status on this issue? It looks like we've manually fixed a lot of session cookie-related issues.
